config/config_test.go

```diff
@@ -252,6 +252,55 @@ func TestReporter(t *testing.T) {
 	assert.Equal(t, "user", cfg.User)
 	assert.Equal(t, "password", cfg.Password)
 
+	//
```

Comment:
Could you please split these into separate, appropriately named tests? There's no reason to keep everything in one function, and it makes it hard to read.

Reply:
done